src/functions/getRequiredStandards.ts

```diff
@@ -15,9 +15,12 @@ export const getRequiredStandards: Handler = async (
   const requiredStandardsService = new RequiredStandardsService(
     requiredStandardsDatabaseService,
   );
+  const euVehicleCategories = event.multiValueQueryStringParameters?.euVehicleCategory;
 
   const defectErrors = validateRequiredStandardsGetQuery(event);
-
+  if (euVehicleCategories && euVehicleCategories.length > 1) {
```

Comment:
Can you add a test reflecting this change for the 400 in requiredStandards.intTest.ts please.